app/views/hosts/_form.html.erb

```diff
@@ -131,6 +136,9 @@
         <%= textarea_f f, :comment, :help_block => _("Additional information about this host"), :size => "col-md-8", :rows => "3", :class => "no-stretch" %>
       </div>
 
+      <% @tabs.values.each do |tab| %>
+        <%= tab %>
```

Comment:
what is in the values? actual HTML?

I think the more suitable way of doing this is to render partials, with values being the partial's path. This will have the added benefit of forcing us to use partials without passing local variables which are code smell...